listeners.go

```diff
@@ -39,28 +40,31 @@ type krazyServer struct {
 
 var (
 	listenersMu sync.Mutex
-	listeners   = make(map[string][]*krazyServer) // by private IP address or unix socket path
+	listeners   = make(map[string][]*krazyServer) // by unix socket path and private IP address
 )
 
-// gokrazyHTTPUnixSocket is the unix socket path that
-// the HTTP server listens on when httpPassword is empty.
-const gokrazyHTTPUnixSocket = "/run/gokrazy-http.sock"
+// GokrazyHTTPUnixSocket is the unix socket path that the HTTP server listens on.
```

Comment:
Can you rename this constant to HTTPUnixSocket please? The package is already named gokrazy, which the current name repeats.